.gitignore

```diff
@@ -166,3 +166,6 @@ tags
 .DS_Store
 # RL pipelines may produce mp4 outputs
 *.mp4
+
+# dependencies
+/transformers
```

Comment:
Is this necessary?

Reply:
Yes think we don't need this, but also ok for me to leave it there